lane.py

```diff
@@ -944,6 +943,7 @@ class Pagination(object):
     DEFAULT_SIZE = 50
     DEFAULT_SEARCH_SIZE = 10
     DEFAULT_FEATURED_SIZE = 10
+    DEFAULT_CRAWLABLE_SIZE = 100
```

Comment:
It takes a lot longer to run a crawlable feed query against Elasticsearch than the materialized view, so I double the size of the feed to compensate.

I could see bumping this up to 200 or even 500, but I don't know how frequently someone importing from a crawlable feed encounters an error and loses a page worth of work.